app/src/main/java/com/ustwo/boilerplate/base/BaseActivity.java

```diff
@@ -0,0 +1,32 @@
+package com.ustwo.boilerplate.base;
+
+import android.os.Bundle;
+import android.support.annotation.LayoutRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+public abstract class BaseActivity<V extends BaseView> extends AppCompatActivity {
+
+  @Override
+  protected void onCreate(@Nullable Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    setContentView(getLayoutId());
+    getPresenter().onViewAttached(getPresenterView());
+  }
+
+  @Override
+  protected void onDestroy() {
+    getPresenter().onViewDetached();
```

Comment:
I can't think that it'd make too much difference. Having an activity in the back stack that is already updated due to being subscribed already seems the better solution to me.

E.g. User navigates from `Activity A` to `Activity B`. `Activity A` receives new data and updates accordingly. User closes `Activity B` and is presented with an already up to date `Activity A`.

If resources become an issue then the Android system will deal with it accordingly by popping an old activity off the back stack.